python/lib/dependabot/python/file_parser.rb

```diff
@@ -48,8 +51,62 @@ def parse
         dependency_set.dependencies
       end
 
+      sig { returns(Ecosystem) }
+      def ecosystem
+        @ecosystem ||= T.let(
+          Ecosystem.new(
+            name: ECOSYSTEM,
+            package_manager: package_manager,
+            language: language
+          ),
+          T.nilable(Ecosystem)
+        )
+      end
+
       private
 
+      def language_version_manager
+        @language_version_manager ||=
+          LanguageVersionManager.new(
+            python_requirement_parser: python_requirement_parser
+          )
+      end
+
+      def python_requirement_parser
+        @python_requirement_parser ||=
+          FileParser::PythonRequirementParser.new(
+            dependency_files: dependency_files
+          )
+      end
+
+      sig { returns(Ecosystem::VersionManager) }
+      def package_manager
+        @package_manager ||= PackageManager.new(python_raw_version, package_manager_requirement)
```

Comment:
Just realized you are using `python_raw_version` for package manager and language. Is there some problem. I believe this is going to be kind of `pip` version or other package manager versions instead of python version? 

I was expecting something like: 
```
PackageManager.new(pip_raw_version)
```

Reply:
@kbukum1 , this is fixed.